### ISSUE TITLE
Issue with the table that has row details and checkboxes.

### ISSUE THREAD

**Issue by daviuz** [bootstrap-vue/bootstrap-vue#2861]
Why does toggling details also toggles the state of the checkbox? Whenever I tick a checkbox and open its details it gets unticked but in the value is still in the v-model.
Then if I tick the unticked checkbox (not closing the details), the data in the v-model gets duplicated.
 
![image](https://user-images.githubusercontent.com/16017297/54505613-74378700-499d-11e9-807a-43f3135de0f2.png)

![image](https://user-images.githubusercontent.com/16017297/54505755-1d7e7d00-499e-11e9-9de7-e2a49050d3e6.png)

**Comment by tmorehouse** [comment]
Could you create a reduced test case on jsfiddle the reproduces the issue? Without knowing your markup or code it is hard to debug/help.

**Comment by tmorehouse** [comment]
Also, are you setting a `primary-key` on your table (that points to a field in your items that would be unique across all items)?

**Comment by tmorehouse** [comment]
Are the checkboxes tied to an external model, or are they tied to a field in your data?

**Comment by daviuz** [comment]
Hi @tmorehouse,

Sorry I didn't have the time to give you a reduced test case but I manage to figure out the solution of the problem. 

My checkboxes were tied to a v-model in the data but the value was pointing to an element of the array that contains an object which doesn't display anything when I tried to do an interpolation. I changed it to point to the `id` property of the object which actually gives me the value.
